app/services/bulk_upload/lettings/row_parser.rb

```diff
@@ -337,7 +356,7 @@ def validate_leaving_reason_for_renewal
   end
 
   def validate_relevant_collection_window
-    return unless start_date && bulk_upload.form
+    return if start_date.blank? || bulk_upload.form.blank?
```

Comment:
I'm with you on not using unless UNLESS it's actually useful, which I agree it isn't here